analytics_data_api/v0/models.py

```diff
@@ -206,3 +208,22 @@ class Video(BaseVideo):
 
     class Meta(BaseVideo.Meta):
         db_table = 'video'
+
+
+if settings.ELASTICSEARCH_LEARNERS_HOST:
+    connections.connections.create_connection(hosts=[settings.ELASTICSEARCH_LEARNERS_HOST])
+
+# By creating the roster index, we can use the @roster.doc_type decorator below
+roster = Index(settings.ELASTICSEARCH_LEARNERS_INDEX)
+
+
+@roster.doc_type
+class RosterEntry(DocType):
+    # pylint: disable=old-style-class
+    class Meta:
+        doc_type = 'roster_entry'
+
+    @classmethod
+    def get_course_user(cls, course_id, username):
+        return cls.search().filter('term', course_id=course_id).query(
```

Comment:
That's interesting.  I would have expected the course ID to be wrapped in a `filter` and the username to nested in a `match` field.  As I understand, the filter component is a caching optimization.  I'm going to punt on that for now and just set these to be both `query` and `term` and the result should be the same.